tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "@vercel/style-guide/typescript",
+  "compilerOptions": {
+    "lib": ["es2020"],
+    "target": "es2020",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
I didn't notice this last time sorry - this is already on in the shared style-guide.

That being said, no real issue with it being redefined here... so only if you're pushing any other changes.

Reply:
I hadn't noticed this before, thanks for pointing it out! Removed it in 083a43c.